source/extensions/filters/http/header_to_metadata/header_to_metadata_filter.cc

```diff
@@ -22,7 +22,7 @@ Config::Config(const envoy::extensions::filters::http::header_to_metadata::v3::C
 
   // don't allow an empty configuration
   if (!response_set_ && !request_set_) {
-    throw new EnvoyException("Must at least specify either response or request config");
+    throw EnvoyException("Must at least specify either response or request config");
```

Comment:
Love this! One of my fuzzers caught this but it's fixed in an open PR :) Anything to fix bugs with tools